apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -51,6 +52,7 @@ function EventTypeSingleLayout({
   team,
   disableBorder,
   enabledAppsNumber,
+  activeAppsNumber,
```

Comment:
I believe web3 apps are not being picked up by these props

![image](https://user-images.githubusercontent.com/65426560/190291304-c2aa3708-e25f-4702-831a-01afd939c4b4.png)